arangod/Agency/State.cpp

```diff
@@ -238,6 +239,11 @@ index_t State::logFollower(query_t const& transactions) {
   VPackSlice slices = transactions->slice();
   size_t nqs = slices.length();
 
+  while(!_ready) {
```

Comment:
is there any chance of a server going up/down quick enough for this to never become "ready" and thus lock this process in an infinite loop ... never letting the process to exit?

Reply:
hawk eye. fixed.